### DISCUSSION TITLE
Any plan or interest in a Wear OS companion app?

### DISCUSSION THREAD

**Discussion by DoubleStrike** [CatimaLoyalty/Android#2039] [Ideas]
This would be really handy to allow display of barcodes directly on the watch display.  Sync could be done via the Android -> Wear APIs.

I would certainly be open to try my hand it, but I am admittedly not a greatly skilled mobile dev.  I have created a couple of Android apps in the past so I am hoping Wear OS development would be similar.

**Comment by TheLastProject** [comment]
Already being discussed in #25 :)

**Reply by DoubleStrike** [reply]
> Already being discussed in #25 :)

I swear I really did do a search but it came up with nothing.

**Reply by TheLastProject** [reply]
No worries, GitHub is silly sometimes :)

I didn't notice you said you were willing to try your hand at it, that would be cool! I originally wanted it as a separate app to "bridge" Catima and WearOS but it's become quite clear nobody is interested in building that so having it as part of Catima might be fine too (as long as we can build two separate flavours, so those who want no proprietary components can still get a "clean" version of Catima). I have no WearOS experience myself but I might grab a cheap WearOS device myself if you get something working to test (my main device is completely deGoogled, but I recently managed to get my hands on a non-deGoogled test device so I could actually test now) :)

**Comment by TheLastProject** [comment]
Implemented in #3218 (without proprietary Google APIs) and released, just waiting for Google approval.
